infra/modules/identity-provider/main.tf

```diff
@@ -0,0 +1,85 @@
+############################################################################################
+## A module for configuring a Cognito User Pool
+## - Configures for email, but not SMS
+## - Configures MFA
+############################################################################################
+
+data "aws_ses_email_identity" "sender" {
+  count = var.sender_email != null ? 1 : 0
+  email = var.sender_email
+}
+
+resource "aws_cognito_user_pool" "main" {
+  name = var.name
+
+  # Use a separate line to support automated terraform destroy commands
```

Comment:
```suggestion
  # We use a separate line here to support automated terraform destroy commands.
  # Do not edit this line, the destroy command looks for it verbatim with `sed`
```

Reply:
All of the lines that get `sed` by destroy commands have the same comment. I'm going to leave this one for now so that it's easier for us to do a comprehensive find and replace in a later PR.